TestAssets/TestProjects/PerformanceTestProjects/SingleTargetGraph/SingleTargetP0/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
I'll add some method calls to make CSC emit references, but dotnet build itself does not care about code dependencies